langserver/workspace_refs.go

```diff
@@ -67,6 +67,7 @@ func (h *LangHandler) handleWorkspaceReferences(ctx context.Context, conn JSONRP
 			}
 		}
 		unvendoredPackages[bpkg.ImportPath] = struct{}{}
+		unvendoredPackages[bpkg.ImportPath+"_test"] = struct{}{}
```

Comment:
I was expecting the main change to be just the `ImportWithTests` like we do in textDocument/refereces. Do you also do this to avoid tricks like `strings.TrimSuffix(pkg, "_test")` that we do in the other reference code?